CHANGE_LOG

```diff
@@ -3,6 +3,10 @@ Version 0.54.0
 
 In development
 
+NOTES:
+
+* Minimum python version raised to 3.7.
```

Comment:
We should do this more often. Leaving notes in the changelog for important changes so we don't have to recall 3mos worth of work at release time.